src/services/facilityServices.js

```diff
@@ -59,6 +59,25 @@ const getFacilityByRegionId = async (id) => {
     return facilities;
 }
 
+const getFacilityByStateId = async (id) => {

```

Comment:
This gets _all_ facilities in a state, right? I think it's better if we use `getFacilitiesByStateId` instead of `getFacilityByStateId`

Reply:
Im following the convention of the the previous engineers there for the naming. getFacilityByRegionId for example is the other method name.

Reply:
I see, well let's scratch the previous conventions then. That was probably a misnomer. Please change it to `getFacilitiesByStateId`

Reply:
AAAAAAAAAAAAAAAAA 










Sure.